knowledge-base/use-reporting-in-dotnetcore-with-entity-framework-core.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Use Telerik Reporting in .Net Core with Entity Data Model
+description: How to use Entity Framework Core for data fetching in .NET Core/Standard projects
+type: how-to
+page_title: Use Telerik Reporting in .Net Core with Entity Framework Core
+slug: use-reporting-in-dotnetcore-with-entity-framework-core
+position: 
+tags: 
+ticketid: 1453347
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>13.0.19.116 and above</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The [EntityDataSource component](../entitydatasource) is dedicated to the Entity Framework and cannot work with Entity Framework Core, 
+although the latter can be used with .NET Framework 4.5+, .NET Core. Framework and .NET Standard - 
+[EF Core Introduction](https://www.entityframeworktutorial.net/efcore/entity-framework-core.aspx).
+
+## Solution
+Use an **ObjectDataSource** component instead. In the latter, you may use the Entity Framework Core to fetch the data and return it 
```

Comment:
Use an **ObjectDataSource** component instead which supports the Entity Framework Core to fetch the data and return it